homeassistant/components/loqed/sensor.py

```diff
@@ -0,0 +1,76 @@
+"""Creates LOQED sensors."""
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    EntityCategory,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import LoqedDataCoordinator, StatusMessage
+from .entity import LoqedEntity
+
+SENSORS: Final[tuple[SensorEntityDescription, ...]] = (
+    SensorEntityDescription(
+        key="ble_strength",
+        name="Bluetooth signal",
```

Comment:
Use `translation_key` and add names to strings.json. See as example https://github.com/home-assistant/core/pull/95732/files

Reply:
Nice, didn't know about that.